packages/react/src/autocomplete-trigger/AutocompleteTrigger.tsx

```diff
@@ -37,3 +79,11 @@ export const AutocompleteTrigger = forwardRef<
 });
 
 AutocompleteTrigger.displayName = "@optiaxiom/react/AutocompleteTrigger";
+
+function forceValueChange(input: HTMLInputElement, value: string) {
```

Comment:
damn - didn't realize we'd need to bring in so much code - now i'm thinking if we should keep re-using SearchInput and just modify SearchInput to support additional `addonAfter` (so more things in addition to its own cross button)

Reply:
I think we can do the update in searchInput vai.